src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable quotes, jsx-quotes, react/prop-types */
+
+import React from 'react';
+import BtnNumbers from './BtnNumbers';
+import BtnOperator from './BtnOperator';
+import Zero from './Zero';
+
+export default function calculator() {
+  return (
```

Comment:
- [ ] It is required that all the components should be class-based components. Kindly change your calculator component from functional to class-based component.

![project-rqt](https://user-images.githubusercontent.com/26597432/211640363-fb2746ca-5415-412d-9641-8af03e1d6c91.png)